Gruntfile.js

```diff
@@ -1030,6 +1032,7 @@ module.exports = function(grunt) {
 
   grunt.registerTask('unit', 'Lint and unit tests', [
     'eslint',
+    'exec:audit-whitelist',
```

Comment:
I don't think we should have this here _and_ in `static-analysis`. IMO it should only be in `static-analysis`. The same goes for `eslint`. I think remove these two from here and then change the description so it's clear it just unit tests and nothing more.

Then `ci-compile` should call `unit` for all its unit testing needs!